### ISSUE TITLE
Feature Request: Allow configuration of other SearchConfiguration options?

### ISSUE THREAD

**Issue by MarcusPope** [profelis/re-search#1]
Hi, I love this extension concept because I'm looking for a way to quickly search a customized subset of a very large mono repo.

If the other search config options were included in the second search, then something like `css: .foo-selector` could be "re-searched" to limit the results to specific css / scss / subfolders.  Or perhaps even feature based selectors: `billing: customer_id`

I'd be happy to submit a pull request if you are busy and you like the idea.

Cheers!

**Comment by profelis** [comment]
Hi! Sorry, I lost you message. Your request is very simple and requires 2-3 lines of code. 

You can add here 
https://github.com/profelis/re-search/blob/master/src/extension.ts#L49
new variable filesToInclude and pass it here https://github.com/profelis/re-search/blob/master/src/extension.ts#L65 similar to isCaseSensitive or matchWholeWord variables. 

I can do this in this weekend but PR sounds better!

**Comment by profelis** [comment]
done in 0.0.5 version

**Comment by MarcusPope** [comment]
I just saw the update and wanted to say thanks for pushing it!  I was actually testing a version I side-loaded earlier today and I have a PR almost ready that I'll push up in a couple more days (limited time of course).  

It includes a couple of other changes that I thought were nice enough and in line with what you have.  And it will include a fix on this version that prevents falsey configuration values from overriding truthy values specified in a previous re-search.  

But otherwise I've already got a dozen rules going that makes searching my codebase so much easier.  Thanks again!
